test/e2e_node/image_list.go

```diff
@@ -66,8 +66,8 @@ var NodeImageWhiteList = sets.NewString(
 // 2. the ones passed in from framework.TestContext.ExtraEnvs
 // So this function needs to be called after the extra envs are applied.
 func updateImageWhiteList() {
-	// Union NodeImageWhiteList and CommonImageWhiteList into the framework image white list.
-	framework.ImageWhiteList = NodeImageWhiteList.Union(commontest.CommonImageWhiteList)
+	// Union NodeImageAllowList and CommonImageWhiteList into the framework image white list.
```

Comment:
@alejandrox1  Should I update `framework image white list` to `framework image pre-pull list` as well?

Reply:
Yes please.